src/types.ts

```diff
@@ -2,33 +2,12 @@ import { LocalVideoTrack, RemoteVideoTrack, TwilioError } from 'twilio-video';
 import { EventEmitter } from 'events';
 
 declare module 'twilio-video' {
-  interface LocalParticipant {
-    setBandwidthProfile: (bandwidthProfile: BandwidthProfileOptions) => void;
-    publishTrack(track: LocalTrack, options?: { priority: Track.Priority }): Promise<LocalTrackPublication>;
-  }
-
-  interface VideoCodecSettings {
-    simulcast?: boolean;
-  }
-
-  interface LocalTrackPublication {
-    setPriority: (priority: Track.Priority) => void;
-  }
-
+  // These help to create union types between Local and Remote VideoTracks
```

Comment:
Should we add this in the sdk that way we can get rid of this declaration?

Reply:
I'm not sure. I'm leaning towards no. This declaration is added because we are creating a union type with LocalVideoTracks and RemoteVideoTracks. Not every application developer may want to do this, so I don't think it needs to be added to the SDK. We aren't adding it here because of a flaw in the SDK types. 

Plus, it isn't taking much space here in `types.js`. I'm not in a hurry to get rid of it 🙂